kernel/src/scan/log_replay.rs

```diff
@@ -165,96 +206,31 @@ impl LogReplayScanner {
         }
     }
 
-    fn get_add_transform_expr(&self) -> Expression {
-        Expression::Struct(vec![
-            column_expr!("add.path"),
-            column_expr!("add.size"),
-            column_expr!("add.modificationTime"),
-            column_expr!("add.stats"),
-            column_expr!("add.deletionVector"),
-            Expression::Struct(vec![column_expr!("add.partitionValues")]),
-        ])
-    }
-
     fn process_scan_batch(
         &mut self,
-        expression_handler: &dyn ExpressionHandler,
+        expression: &dyn ExpressionEvaluator,
```

Comment:
nit: `expression` implies this is a predicate. Perhaps `expression_evaluator`, but your call :)

Reply:
Renamed it to say (conceptually) what it is: `add_transform`

Reply:
update: renamed the wrong one (or rather, renamed a different good one)

I renamed _this one_ as `predicate`

Reply:
... and renamed back to `add_transform` because it's _not_ a predicate at all